api/swagger.yml

```diff
@@ -2681,6 +2681,11 @@ paths:
           description: limit the number of items in return to 'amount'. Without further indication on actual number of items.
           schema:
             type: boolean
+        - in: query
+          name: firstParent
```

Comment:
from a quick search all the query parameters are snake case (except storageClass)